Client/Assets/suggestedsites.json

```diff
@@ -1 +1 @@
-[{"url": "https://www.mozilla.org/en-US/", "bgcolor": "0xce4e41", "imageurl": "asset://suggestedsites_mozilla", "trackingid": 632, "title": "The Mozilla Project"}, {"url": "https://support.mozilla.org/en-US/products/mobile", "bgcolor": "0xf37c00", "imageurl": "asset://suggestedsites_fxsupport", "trackingid": 631, "title": "Firefox Help and Support"}]
```

Comment:
Maybe run this through a JSON formatter so that it is not one huge line that is difficult to diff?